crates/example_dialogue_view/src/setup.rs

```diff
@@ -31,77 +31,70 @@ fn setup(mut commands: Commands) {
     commands
         .spawn((
             fmt_name("root"),
-            NodeBundle {
-                style: Style {
-                    display: Display::Grid,
-                    width: Val::Percent(100.0),
-                    height: Val::Percent(100.0),
-                    align_content: AlignContent::End,
-                    justify_content: JustifyContent::SpaceAround,
-                    grid_auto_flow: GridAutoFlow::Row,
-                    grid_template_columns: vec![RepeatedGridTrack::minmax(
-                        1,
-                        MinTrackSizingFunction::Auto,
-                        MaxTrackSizingFunction::Px(DIALOG_WIDTH),
-                    )],
-                    // In the web, `GridTrack::auto()` should already work like this,
-                    // but it seems like Bevy disagrees?
-                    // If you remove this line in the future and everything looks the same, go ahead!
-                    grid_auto_rows: vec![GridTrack::min_content()],
-                    ..default()
-                },
-                visibility: Visibility::Hidden,
+            Node {
+                display: Display::Grid,
+                width: Val::Percent(100.0),
+                height: Val::Percent(100.0),
+                align_content: AlignContent::End,
+                justify_content: JustifyContent::SpaceAround,
+                grid_auto_flow: GridAutoFlow::Row,
+                grid_template_columns: vec![RepeatedGridTrack::minmax(
+                    1,
+                    MinTrackSizingFunction::Auto,
+                    MaxTrackSizingFunction::Px(DIALOG_WIDTH),
+                )],
+                // In the web, `GridTrack::auto()` should already work like this,
+                // but it seems like Bevy disagrees?
+                // If you remove this line in the future and everything looks the same, go ahead!
+                grid_auto_rows: vec![GridTrack::min_content()],
```

Comment:
I didn't touch this, but for reference, this is what  happens when it's removed: 
![image](https://github.com/user-attachments/assets/62c255e6-f3c9-429d-8a7b-1b19c5c6d0dd)

Reply:
Pinging @bash, who discovered this, in case they want to file a bug :)

Reply:
I think back then I didn't report a bug because I wasn't sure if it actually was one :) I have an intuition for how grids behave based on what browsers do, but I don't know the spec ^^